keras_nlp/layers/random_swap_test.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for RandomSwaps Layer."""
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers import RandomSwap
+from keras_nlp.tokenizers import UnicodeCodepointTokenizer
+
+
+class RandomSwapTest(tf.test.TestCase):
+    def test_shape_and_output_from_word_swap(self):
+        keras.utils.set_random_seed(1337)
+        inputs = ["Hey I like", "Keras and Tensorflow"]
+        split = tf.strings.split(inputs)
+        augmenter = RandomSwap(rate=0.7, max_swaps=3, seed=42)
+        augmented = augmenter(split)
+        output = tf.strings.reduce_join(augmented, separator=" ", axis=-1)
+        self.assertAllEqual(output.shape, tf.convert_to_tensor(inputs).shape)
+        exp_output = [b"like I Hey", b"Tensorflow Keras and"]
```

Comment:
could we wrap all these expected outputs in convert to tensor to avoid the bytestrings?